machineconfiguration/v1/types.go

```diff
@@ -100,6 +100,11 @@ type ControllerConfigSpec struct {
 	// +nullable
 	InternalRegistryPullSecret []byte `json:"internalRegistryPullSecret"`
 
+	// InternalAPICert is used to generate and update /etc/kubelet/kubeconfig
+	// +kubebuilder:validation:Required
+	// +nullable
+	InternalAPICert []byte `json:"internalAPICert"`
```

Comment:
the CA bundle to use is https://github.com/openshift/api/tree/master/tls/docs/kube-apiserver%20Serving%20Certificates#kube-apiserver-total-serving-ca located at `-n openshift-config-managed 	configmap/kube-apiserver-server-ca`